MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -843,6 +844,29 @@ public void DrawIndexedPrimitives(PrimitiveType primitiveType, int baseVertex, i
             PlatformDrawIndexedPrimitives(primitiveType, baseVertex, startIndex, primitiveCount);
         }
 
+        public void DrawIndexedPrimitives(PrimitiveType primitiveType, int baseVertex, int startIndex, int primitiveCount)
+        {
+            if (_vertexShader == null)
+                throw new InvalidOperationException("Vertex shader must be set before calling DrawIndexedPrimitives.");
+
+            if (_vertexBuffer == null)
+                throw new InvalidOperationException("Vertex buffer must be set before calling DrawIndexedPrimitives.");
+
+            if (_indexBuffer == null)
+                throw new InvalidOperationException("Index buffer must be set before calling DrawIndexedPrimitives.");
+
+            if (primitiveCount <= 0)
+                throw new ArgumentOutOfRangeException("primitiveCount");
+
+            unchecked
+            {
+                _graphicsMetrics._drawCount++;
+                _graphicsMetrics._primitiveCount += (ulong)primitiveCount;
+            }
+
+            PlatformDrawIndexedPrimitives(primitiveType, baseVertex, startIndex, primitiveCount);
+        }

```

Comment:
The `minVertexIndex` and `numVertices` parameters were used in XNA, but were ignored by Nvidia drivers and required in some cases by ATI drivers.  This was the cause of some primitives not drawing on ATI cards only.